### ISSUE TITLE
🐛 [Bug] Violet Tragan deals too much damage.

### ISSUE THREAD

**Issue by Daribon** [vmangos/core#1609]
## 🐛 Bug report

If you eat Violet Tragan (itemID 8526) it will deal vastly more damage than supposed.
It will tick 10 damage several times per second.
![WoW_zdhDOM0BK4](https://user-images.githubusercontent.com/6137576/188773901-0e428429-de6f-4159-93d8-b8dc5f268305.png)
It dealt over 2000 damage on me when I used it.
![WoW_UsJQSHo8UY](https://user-images.githubusercontent.com/6137576/188773992-cd617be2-576c-4f36-8912-d0c8738ef925.png)

### Expected behavior

It should only tick every 5 seconds. Vastly slower than how it currently does.
Image taken classic:
![nomacs_XXIFAJ4YH9](https://user-images.githubusercontent.com/6137576/188774353-07ca7224-9a25-4ee5-ad3f-6ef8c3ec2704.png)

Allakhazam comment from patch 1.4.2 confirms that the tick is accurate from classic:
![firefox_LWk4B49jfd](https://user-images.githubusercontent.com/6137576/188774677-d6f4a875-6932-49bc-b399-3d8562cd04df.png)

Thottbot comment from patch 1.2.4 confirms it as well:
![firefox_g0xEQLfRTa](https://user-images.githubusercontent.com/6137576/188774972-62a6e91c-3857-4326-995f-feebe621f912.png)

### Steps to reproduce

1. .additem 8526
2. Eat it
3. Watch as it tick 10 damage several times per second.

### Version & Environment

Client Version: 1.12.1.5875

Commit Hash:  https://github.com/vmangos/core/commit/373fde44b3c8c6f5236656e31dabffbb5f728037

OS Client: Microsoft Windows 11 Pro N 10.0.22000 Build 22000
OS Server: Microsoft Windows 11 Pro N 10.0.22000 Build 22000

### Crashlog

- None

**Comment by ratkosrb** [comment]
This spell has been changed in classic in patch 1.13.6.36611. Notice that EffectAuraPeriod changes from 0 to 5000.

Before:
https://wow.tools/dbc/?dbc=spelleffect&build=1.13.6.36524#page=1&search=684162

After:
https://wow.tools/dbc/?dbc=spelleffect&build=1.13.6.36611#page=1&search=684162

**Comment by Daribon** [comment]
Comment from 1.12.1 says it did no damage:
![firefox_h0d022DWlo](https://user-images.githubusercontent.com/6137576/188781308-7247b79e-6b3b-4782-9895-06730a545d75.png)

Another comment from 1.12.1 says it did damage:
![firefox_yF0KLO43HV](https://user-images.githubusercontent.com/6137576/188782098-e5c28a2b-aa29-44a6-b6ea-41c0764b6852.png)

A comment from early TBC states that it did no damage:
![firefox_GcHFxnbVaB](https://user-images.githubusercontent.com/6137576/188781875-69d57f69-4a4f-45e4-bbb2-a1e41393f097.png)

Another comment from early classic states that it didn't do any damage and it didn't show up in combat log.
![firefox_p5OXJfzauY](https://user-images.githubusercontent.com/6137576/188781072-41707a8a-1c7d-4120-a96b-2bc48e7a0055.png)

Might have been broken in late vanilla causing it to deal no damage and because classic using 1.12.0 data it too was in early classic until they fixed it.

It is whoever weird that there is one comment stating no damage and another stating damage and both being from 1.12.1.

TL;DR:
It either dealt 10 damage per 5 seconds or did no damage in 1.12.1.
Either way the massive damage per tick is not blizzlike.
